src/resolvers/userResolvers.ts

```diff
@@ -10,6 +10,7 @@ const userResolvers = {
     // login
     logIn: (parent:any, args:any, ctx:any) => User.logIn(parent, args, ctx),
     verifyEmail: (parent:any, args:any, ctx:any) => User.verifyEmail(parent, args, ctx),
+    updateUser: (parent:any,args:any,ctx:any) => User.updateUser(parent,args,ctx),
   }
 };
```

Comment:
this should give esllint error if you are not getting it  it means ur eslint is not active so make sure you work on it